service_container/service_closures.rst

```diff
@@ -0,0 +1,112 @@
+.. index::
+    single: DependencyInjection; Service Closures
+
+Service Closures
+================
+
```

Comment:
```suggestion

.. versionadded:: 5.4

    Service closures were introduced in Symfony 5.4.

```

Reply:
@OskarStark Service closures were introduced in version 3.3, only the `service_closure()` DSL function was added in 5.4

Reply:
Ok, please adjust the sentence accordingly. Thanks

Reply:
Added